Makefile

```diff
@@ -219,8 +249,9 @@ e2e-test:
 	ginkgo -v --slowSpecThreshold=10 test/e2e
 
 e2e-dependencies:
-	go get github.com/onsi/ginkgo/ginkgo@v1.14.1
-	go get github.com/onsi/gomega/...@v1.10.1
+	go get github.com/onsi/ginkgo/ginkgo@v1.16.4
+	go get github.com/onsi/gomega/...@v1.13.0
+	go get github.com/open-cluster-management/governance-policy-propagator@v0.0.0-20211012174109-95c3b77cce09
```

Comment:
Could you please change this to a `-io` reference?

Reply:
Go modules are going to be hard for this for now. For example, since the -io propagator declares its module in its go.mod **without** -io, Go gives this error when trying to import it:

```
go: github.com/open-cluster-management-io/governance-policy-propagator@v0.5.0: parsing go.mod:
	module declares its path as: github.com/open-cluster-management/governance-policy-propagator
	        but was required as: github.com/open-cluster-management-io/governance-policy-propagator
```

So, for now, I want to defer that change.